activerecord/test/cases/schema_dumper_test.rb

```diff
@@ -7,6 +7,7 @@ class SchemaDumperTest < ActiveRecord::TestCase
 
   setup do
     ActiveRecord::SchemaMigration.create_table
+    @connection = ActiveRecord::Base.connection
```

Comment:
We set an ivar here, act on it in stub_version, but never use it in a test. Chop it from setup and pass it to stub_version.